cli/internal/cmd/iamcreateaws.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edgelesssys/constellation/v2/cli/internal/cloudcmd"
+	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
+	"github.com/spf13/cobra"
+)
+
+// newIAMCreateAWSCmd returns a new cobra.Command for the iam create aws command.
+func newIAMCreateAWSCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "aws",
+		Short: "Create IAM configuration on AWS for your Constellation cluster",
+		Long:  "Create IAM configuration on AWS for your Constellation cluster.",
+		Args:  cobra.ExactArgs(0),
+		RunE:  runIAMCreateAWS,
+	}
+
+	cmd.Flags().String("prefix", "", "Name prefix for all resources.")
+	must(cobra.MarkFlagRequired(cmd.Flags(), "prefix"))
+	cmd.Flags().String("zone", "", "AWS availability zone the resources will be created in (e.g. us-east-2a). Find available zones here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones. Note that we do not support every zone / region. Find a list of currently supported regions here: https://docs.edgeless.systems/constellation/next/getting-started/first-steps.")
```

Comment:
```suggestion
	cmd.Flags().String("zone", "", "AWS availability zone the resources will be created in (e.g. us-east-2a). Find available zones here: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones. Note that we do not support every zone / region. You can find a list of all supported regions in our docs.")
```
I'm on the fence about this. On the one hand we shouldn't directly link to docs, especially to the "next" version of them, since the docs can and will change rapidly. On the other hand, it is nice to dive the user a direct hint. 

Since the user should follow the docs anyway when creating a Constellation, I think that it is ok to generally refer them to the docs.